theme/less/components/discussions.less

```diff
@@ -217,125 +217,18 @@ You can use a discussion creation button too.
 
```

Comment:
I think you need to update the discussions stylemark examples.

Reply:
True ! We can also move it back to vue files when https://github.com/etalab/udata-front/pull/33 is merged